### PR TITLE
JBIDE-21323 EL Knowledge Base Problem with enums

### PR DESCRIPTION
**Pull request by scabanovich**

### CODE REVIEW

File: common/tests/org.jboss.tools.common.el.core.test/src/org/jboss/tools/common/el/core/test/resolver/TypeInfoCollectorTest.java [hunk 1]
```diff
@@ -132,6 +132,19 @@ public void testProperties() throws CoreException {
...
+		IType bean = jp.findType(className);
+		TypeInfoCollector.TypeInfo typeInfo = new TypeInfoCollector.TypeInfo(bean, null, false);
+		TypeInfoCollector collector = typeInfo.getTypeCollector(false, false);
+		MemberInfo status = getMethod(collector, "getStatus");
+		assertNotNull(status);
+		collector = status.getTypeCollector(false, false);
+		MemberInfo name = getMethod(collector, "name");
+		assertNotNull(name);
+	}
+
```
Comment (andrei-ivanov) [inline common/tests/org.jboss.tools.common.el.core.test/src/org/jboss/tools/common/el/core/test/resolver/TypeInfoCollectorTest.java]: What about SignOffStatus#getLabel()?

Comment (scabanovich) [inline common/tests/org.jboss.tools.common.el.core.test/src/org/jboss/tools/common/el/core/test/resolver/TypeInfoCollectorTest.java]: It should work as property 'label'. Does it not?

Comment (andrei-ivanov) [inline common/tests/org.jboss.tools.common.el.core.test/src/org/jboss/tools/common/el/core/test/resolver/TypeInfoCollectorTest.java]: Ah, sorry, seems I've copy/pasted the wrong message in the issue. I've updated the issue now.
The point of the 2nd example was to show that even a standard property (label) isn't detected in some cases.

**Comment by alexeykazakov** [comment]
Pushed to master and 4.3.x

**Pull request closed without merging**